client/src/services/chatService.js

```diff
@@ -0,0 +1,16 @@
+// from app.models.chat import ChatRequest, ChatResponse
+export const generateChatResponse = async (chatRequest) => {
+    const url = "http://127.0.0.1:8000/chat/generate"
```

Comment:
import from endpoint.js

Reply:
importing from endpoint gives errors because the endpoint file is not inside src/